### DISCUSSION TITLE
Look into changing the design of note boxes on lightmode

### DISCUSSION THREAD

**Discussion by Asartea** [TheOdinProject/theodinproject#3947] [Ideas]
Complete the following REQUIRED checkboxes:

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
-   [x] The title of this issue follows the `brief description of request` format, e.g. `Add dark mode to website`

<hr>

**1. Description of the Feature Request:**
Converting over from #3915:

Should the background color of the notice boxes on lightmode be increased to something that contrasts more clearly with the background? Alternatively, should any other design changes be made to them? @ManonLef suggested:

> I think if it were to have no background color it could look good if the left border wasn't shaped with rounding. The rounding makes it seem as if it should wrap around something.

**2. Acceptance Criteria:**

```[tasklist]
### Acceptance Criteria
- [ ] A decision on reached on if/which changes are needed
- [ ] If needed, any necessary changes are implemented
```

**Comment by CouchofTomato** [comment]
You've been tagged here @ManonLef so is it something you want to respond to?

**Comment by ManonLef** [comment]
Thanks @CouchofTomato 
Not really, I think I added my share in the linked issue :)
